requirements.txt

```diff
@@ -1,16 +1,16 @@
-awkward~=2.0.0
+awkward~=2.1.0
 boost-histogram~=1.3.0
 decaylanguage~=0.15.0
 hepstats~=0.6.0
 hepunits~=2.3.0
 hist~=2.6.0
 histoprint~=2.4.0
-iminuit~=2.18.0
-matplotlib>2.0.0
-mplhep~=0.3.0
+iminuit~=2.21.0
+matplotlib>3.4.0
+mplhep~=0.3.26
```

Comment:
```suggestion
mplhep~=0.3.0
```

Reply:
Indeed we make it fully compatible with patch versions as mentioned on the README.